apps.yaml

```diff
@@ -1,4 +1,18 @@
 ---
+diffusion-electrolyte-mixture:
+    metadata:
+        title: Diffusion_Electrolyte_Mixture
```

Comment:
Could you use blank spaces or dashes instead of underscores in the title? Suggestion: Diffusion Electrolyte Mixture.

Reply:
Thanks. Underscore removed.